webpack.config.js

```diff
@@ -75,7 +75,17 @@ const babelLoader = {
 	},
 };
 
-module.exports = function( env, argv ) {
+/**
+ * Return a webpack config object
+ *
+ * @see {@link https://webpack.js.org/configuration/configuration-types/#exporting-a-function}
+ *
+ * @param  {object}  env                environment options
+ * @param  {string}  env.extensionName  set by bin/sdk/gutenberg.js when building Gutenberg extensions
```

Comment:
the description here may be overly specific. I think for the purposes of this file it's more like "set to add 'prefix class namespace' to every built CSS rule"